packages/block-library/src/gallery/edit.js

```diff
@@ -87,7 +89,7 @@ class GalleryEdit extends Component {
 
 	onSelectImages( images ) {
 		this.props.setAttributes( {
-			images: images.map( ( image ) => pick( image, [ 'alt', 'caption', 'id', 'link', 'url' ] ) ),
+			images: images.map( ( image ) => pick( image, RELEVANT_MEDIA_FIELDS ) ),
```

Comment:
You could export a method out of `( image ) => pick( image, RELEVANT_MEDIA_FIELDS )`:

```js
export const piclRelevantMediaFiles = ( image ) => pick( image, RELEVANT_MEDIA_FIELDS );
```

and also use it inside this file.